RoboJackets/robocup-software
# Basic defensive kick

## Description
A basic defensive kick restart (Ball on our side) that chips the ball up field.

## Associated Issue
Issue #1408 

## Steps to test
1. Setup a restart on our half of the field
2. Run the play

Expected result: Two of the robots should move up the field, and the passing robot should chip to the robot directly up the field from it.
